bench/bench.ml

```diff
@@ -30,7 +30,38 @@ let plot metrics max =
     done ; Fmt.pr "\n%!" ;
   done
 
-let inflate max =
+type result = {
+  time : int;
+  in_bytes_per_sec : int;
+  out_bytes_per_sec : int;
+  live_heap_bytes : int;
+}
+[@@deriving to_yojson]
```

Comment:
The `dune` file should be upgraded according to this new dependency. I'm not a big fan to use PPX but the tool is out of the distribution, so it should be fine.

Reply:
> the tool is out of the distribution

And this is why the CI works even if you did not upgrade the `dune` file.